app/views/shared/_messages.html.slim

```diff
@@ -2,4 +2,4 @@
   - flash.each do |name, msg|
     - if msg.present? && msg.is_a?(String)
       div class="alert alert-#{name}" role='alert'
-        = msg
+        = sanitize(msg)
```

Comment:
It worked for me after pulling down this branch. Did you add `.html_safe`, like this: `safe_join([msg.html_safe])`?